doc/nrf/device_guides/working_with_fem.rst

```diff
@@ -478,6 +492,17 @@ To use the Simple GPIO implementation of FEM with SKY66112-11, complete the foll
 
    The state of the other control pins should be set according to the SKY66112-11 documentation.
    See the official `SKY66112-11 page`_ for more information.
+#. On nRF53 devices, you must also apply the same devicetree node mentioned in step 1 to the network core.
+   To do so, apply the overlay to the correct network core child image by creating an overlay file named :file:`child_image/*childImageName*.overlay` in your application directory, for example :file:`child_image/multiprotocol_rpmsg.overlay`.
+
+   The ``*childImageName*`` string must be one of the following values:
+
+   *  ``multiprotocol_rpmsg`` for multiprotocol applications having support for both 802.15.4 and Bluetooth.
+   *  ``802154_rpmsg`` for applications having support for 802.15.4, but not for Bluetooth.
+   *  ``hci_ipc`` for application having support for Bluetooth, but not for 802.15.4.
+
+   .. note::
+       This step is not needed when testing with :ref:`direct_test_mode` and :ref:`radio_test` on the nRF53 Series devices.
```

Comment:
"On nRF53 devices, you must also apply the same devicetree node mentioned in step 1 to the network core" ... 
What about writing directly in step 1 that the 

```
1. Add the following node in the devicetree file that is applied to the SoC core containing the RADIO peripheral:
   .. code-block::
      / {
         nrf_radio_fem: name_of_fem_node {
            compatible = "skyworks,sky66112-11", "generic-fem-two-ctrl-pins";
            ctx-gpios = <&gpio0 13 GPIO_ACTIVE_HIGH>;
            crx-gpios = <&gpio0 14 GPIO_ACTIVE_HIGH>;
         };
      };
```
Then you could explain how to do this for nRF5340 and all those child images stuff.

Simillar should be done for nrf21540.

Reply:
discussed f2f, keep it as is